charm/trusty/calico-docker/Makefile

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/make
+
+build: virtualenv lint test
+
+virtualenv: .venv/bin/python
+.venv/bin/python:
+	sudo apt-get install -y  python-virtualenv
+	virtualenv .venv
+	.venv/bin/pip install nose flake8 mock pyyaml charmhelpers charm-tools ecdsa ansible ansible-lint
+
+lint: .venv/bin/python
+	@.venv/bin/flake8 hooks
+	@.venv/bin/ansible-lint playbooks/site.yaml
+	@.venv/bin/charm proof
+
+unit_test: .venv/bin/python
+	@echo Starting tests...
+	@CHARM_DIR=. PYTHONPATH=./hooks .venv/bin/nosetests -vv --nologcapture unit_tests
```

Comment:
This target doesn't work! I suspect the reason for that is that this charm doesn't actually *have* any unit tests. I'm not too bothered by that, but we shouldn't have a makefile target that doesn't correspond to anything... ;)